src/chttpd/src/chttpd_cors.erl

```diff
@@ -280,7 +280,7 @@ allow_credentials(Config, Origin) ->
 get_cors_config(#httpd{cors_config = undefined, mochi_req = MochiReq}) ->
     Host = couch_httpd_vhost:host(MochiReq),
 
-    EnableCors = config:get("httpd", "enable_cors", "false") =:= "true",
+    EnableCors = chttpd_util:get_chttpd_config_boolean(enable_cors, false) =:= true,
```

Comment:
It might be simpler to use just `chttpd_util:get_chttpd_config_boolean(enable_cors, false)` I think?